docs/userguide.rst

```diff
@@ -115,6 +115,21 @@ file of the analysis, by just running:
 
 Note that downloading directories is not yet supported.
 
+Running example
+~~~~~~~~~~~~~~~
+
+If you have fully working example with reana.yaml, you can run the workflow
+easily via the `run` wrapper command, which will create a new workflow, upload
```

Comment:
- "Running analysis"

- "fully working analysis"

- ``reana.yaml`` in double backquotes

- ``run`` should be enclosed in double backquotes, not single backquotes, in reST

- add `$ # ... ` or something after `status` command to signify that workflow runs